### PR TITLE
#167127538  Modify the user signup data

### PR DESCRIPTION
**Pull request by Chiagozielam**
#### What does this PR do?
- Modify the user signup data to remove redundant input fields during user signup.

#### Description of Task to be completed?
- delete object with a name key and a value of firstname from api.js in docs
- delete firstName and lastName fields from our userObject in users controller.js file
- delete firstName and LastName fields from our stringParams object in users middleware.js
- delete firstName field from our userRequestObject object

#### What are the relevant pivotal tracker stories?
[#167127538](https://www.pivotaltracker.com/story/show/)

#### Questions:

### CODE REVIEW

File: src/app/user/controller.js [hunk 1]
```diff
@@ -16,12 +16,10 @@ const createTokenPayload = user => {
 };
 
 const createUser = async (req, res) => {
-  const { email, username, password, firstName, lastName, role } = req.body;
+  const { email, username, password, role } = req.body;
```
Comment (ebenezerdon) [inline src/app/user/controller.js]: Please remove `role` from the request object. 
@Chiagozielam

Comment (Chiagozielam) [reply src/app/user/controller.js]: Okay, I'll do this ASAP

Comment (ebenezerdon) [reply src/app/user/controller.js]: Please add role back here. I forgot I'd put a condition for the test environment.

Your PR should be good to go after this.

File: src/app/user/controller.js [hunk 2]
```diff
@@ -16,12 +16,10 @@ const createTokenPayload = user => {
 };
 
 const createUser = async (req, res) => {
-  const { email, username, password, firstName, lastName, role } = req.body;
+  const { email, username, password } = req.body;
```
Comment (ebenezerdon) [inline src/app/user/controller.js]: Please add role back here @Chiagozielam

Comment (Chiagozielam) [reply src/app/user/controller.js]: okay, I'm just trying to figure out how to add this without pushing my other staged changes

Comment (ebenezerdon) [reply src/app/user/controller.js]: Is this not in a different branch?

**Pull request closed without merging**